ironrdp-cli/src/gui.rs

```diff
@@ -0,0 +1,154 @@
+use std::fmt::Debug;
```

Comment:
This code appears to be now unused in `ironrdp-cli` as well.

Reply:
fixed